### PR TITLE
Selenium: Add changes to the 'RenameParametersTest' to incease reliability

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Add some changes to the _checkRenameParameters15()_ method in the _RenameParametersTest_
to increase a reliability when runs on CI

### What issues does this PR fix or reference?
#11729

### CODE REVIEW

**Comment by artaleks9** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/parameters/RenameParametersTest.java [hunk 1]
```diff
@@ -199,9 +199,12 @@ public void checkRenameParameters15() throws Exception {
     events.clearAllMessages();
     editor.waitActive();
     editor.goToCursorPositionVisible(15, 23);
-    editor.launchLocalRefactor();
-    editor.typeTextIntoEditor("i");
-    editor.typeTextIntoEditor(Keys.ENTER.toString());
+    editor.launchRefactorForm();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/parameters/RenameParametersTest.java]: Could you, please, divide separate steps by empty rows and add short comment on the top of steps with explanation what the steps are aimed to do?

Comment (artaleks9) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/parameters/RenameParametersTest.java]: OK, I'll correct

**Review by dmytro-ndp — APPROVED** [review]

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1105/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1105//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11766
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**